src/Json/Encode.elm

```diff
@@ -55,6 +55,8 @@ int =
     Native.Json.identity
 
 
+{-| Encode a Float. `Infinity` and `NaN` are encoded as `null`.
+-}
```

Comment:
Shouldn't Elm's floats map on to the same floats though? Like, why would NaN not result in Nan? Is that not valid JSON or something?